### DISCUSSION TITLE
How do I change the decimal setting, the thousands separator?

### DISCUSSION THREAD

**Discussion by lscapim** [getredash/redash#6471] [General]
Hi,
We need to be able to use the space our point as the thousands separator and the comma as the decimal separator.

1,200.50 vs 1 200,50 (1.200,50) is quite different if you are used to either format.

Is there anything planned for this feature? Can we expect to see you on Redash soon?

**Comment by lscapim** [comment]
Can someone help me?

**Comment by guidopetri** [comment]
Hey @lscapim , I am not sure whether this is supported yet. We have an i18n PR in the works #933 / #6006 though.
